grakn-engine/src/test/java/ai/grakn/engine/controller/util/JsonConceptBuilder.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.engine.controller.util;
+
+import ai.grakn.engine.controller.response.Attribute;
+import ai.grakn.engine.controller.response.AttributeType;
+import ai.grakn.engine.controller.response.Concept;
+import ai.grakn.engine.controller.response.Entity;
+import ai.grakn.engine.controller.response.EntityType;
+import ai.grakn.engine.controller.response.MetaConcept;
+import ai.grakn.engine.controller.response.Relationship;
+import ai.grakn.engine.controller.response.RelationshipType;
+import ai.grakn.engine.controller.response.Role;
+import ai.grakn.engine.controller.response.Rule;
+import ai.grakn.util.Schema;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import mjson.Json;
+
+import java.io.IOException;
+
+/**
+ * <p>
+ *     Takes a string representing a {@link ai.grakn.engine.controller.response.Concept} and serialises it into
+ *     it's correct object
+ * </p>
+ *
+ * @author Filipe PEliz Pintio Teixeira
```

Comment:
The hooves must make it difficult to type 😞

Reply:
Don't judge centaurs!